wiki/Tournaments/OWC/2023/zh-tw.md

```diff
@@ -0,0 +1,426 @@
+---
+tags:
+  - OWC
+  - OWC2023
+---
+
+# osu! World Cup 2023
+
+![OWC 2023 banner](img/owc2023-banner.jpg)
+
+**osu! World Cup 2023** (***OWC 2023***) 是一個以國家為隊伍的 osu! 世界賽，由 [osu! team](/wiki/People/osu!_team) 主辦，這次為該比賽的第十四屆。
+
+## 比賽時程
+
+| 內容 | 日期 |
+| --: | :-- |
+| 報名 | 2023 年 9 月 21 日 至 2023 年 10 月 5 日 (23:59 UTC) |
+| 預選賽圖池公布 | 2023 年 10 月 15 日 (14:00 UTC) |
+| 預選賽 | 2023 年 10 月 21 日 / 2023 年 10 月 22 日 |
+| 32強賽 | 2023 年 10 月 28 日 / 2023 年 10 月 29 日 |
+| 16強賽 | 2023 年 11 月 4 日 / 2023 年 11 月 5 日 |
+| 8強賽 | 2023 年 11 月 11 日 / 2023 年 11 月 12 日 |
+| 準決賽 | 2023 年 11 月 18 日 / 2023 年 11 月 19 日 |
+| 決賽 | 2023 年 11 月 25 日 / 2023 年 11 月 26 日 |
+| 總決賽 | 2023 年 12 月 2 日 / 2023 年 12 月 3 日 |
+
+## 獎項
+
+主辦方提供美金 $5,000 的獎金以及限定獎品。
+
+| 名次 | 獎賞 |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "冠軍") | 48% 的獎金、限量發行獎品、個人頁面獎章、一年的 **osu! Champion** 個人稱號 |
+| ![Silver crown](/wiki/shared/crown-silver.png "亞軍") | 32% 的獎金、限量發行獎品、個人頁面獎章 |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "季軍") | 20% 的獎金、限量發行獎品、個人頁面獎章 |
```

Comment:
likewise, gold/silver/bronze crown should be translated

Reply:
still need this one :eyes:

Reply:
My bad, fixed. :)